web/src/main/resources/assets/js/main-template.js

```diff
@@ -276,8 +277,8 @@ $(document).ready(function (e) {
                         endIndex = endIndex < 0 ? cleanedText.length : endIndex;
                         var wordUnderCursor = cleanedText.substring(startIndex, endIndex);
                         if(this.selectionStart == 0 || this.value.substr(this.selectionStart - 1, 1) === "\n") {
-                           document.getElementById("ev_value").innerHTML = "<b>root:</b> speed_factor, priority, max_speed, max_speed_fallback, distance_influence, areas";
-                        } else if(wordUnderCursor === "priority" || wordUnderCursor === "speed_factor" || wordUnderCursor === "max_speed") {
+                           document.getElementById("ev_value").innerHTML = "<b>root:</b> priority, speed, distance_influence, areas";
+                        } else if(wordUnderCursor === "priority" || wordUnderCursor === "speed") {
```

Comment:
Hm, the autocomplete feature does not work very well currently. For example we get no hint to type if/else/else if and after we do we do not get a hint to type an encoded value name. This could probably be improved without too much effort (but great help for the user), but we should optimize for one format only? Not sure if that should be yaml or json and if yaml it could be the 'one-line-per-statement' format or the more verbose one...

Reply:
And by the way: Should we rename the 'flex' button or otherwise make it more visible?

Reply:
For now I would postpone the improvement of the auto complete feature (except this can be solved quickly :) ? )

Reply:
I think it can be improved with a few simple rules if we e.g. assume the explicit yaml format, but its hard to make it work for all formats at the same time probably. But also we can postpone this. I just thought if there are users who are willing to start experimenting with the new custom model language we should probably do our best they have a good experience and the autocomplete makes it much more comfortable to use IMO. Same with proper syntax checking and detailed error messages, but this is another story maybe.

Reply:
> assume the explicit yaml format, but its hard to make it work for all formats at the same time probably.

Yes, I would assume yaml for the UI. And if JSON is copy pasted into the box it could be converted to YAML and we can also provide a "convert to YAML" feature.

Reply:
Ok yes. And do you want to change the docs to json?

Reply:
At the moment I slightly favor JSON over YAML for the documentation as we can assume more knowledge from users that do server-side profiles. But maybe we decide this in #2232?